modules/video/src/lapjv/lapjv.cpp

```diff
@@ -0,0 +1,339 @@
+// This file is part of OpenCV project.
+// It is subject to the license terms in the LICENSE file found in the top-level directory
+// of this distribution and at http://opencv.org/license.html.
```

Comment:
The file is part of LAPJV. Please do not add OpenCV legal header, but presume the original text.